network-api/networkapi/buyersguide/templates/category_page.html

```diff
@@ -15,11 +15,11 @@ <h1 class="h3-heading">{{ category.name }}</h1>
       {% endif %}
     </div>
 
-    <div class="categories-container">
+    <div class="d-flex flex-wrap row">
       {% for product in products %}
-      <div class="category-item-container{% if product.draft %} draft-product{% endif %}{% if product.adult_content %} adult-content{% endif %}">
-        <a class="" href="/en/privacynotincluded/products/{{ product.slug }}">
-          <div class="category-image-container">
+      <div class="category-item-container col-6 text-left {% if product.draft %} draft-product{% endif %}{% if product.adult_content %} adult-content{% endif %}">
+        <a class="text-decoration-none text-reset" href="/en/privacynotincluded/products/{{ product.slug }}">
```

Comment:
this doesn't seem to do anything 😬 

<img width="439" alt="image" src="https://user-images.githubusercontent.com/2896608/52388204-34968a80-2a42-11e9-8171-c74b90bf2efd.png">

Reply:
and `position-relative` and `overflow-hidden` too

Reply:
removed this.